### PR TITLE
[ci] add manual developer triggered update

### PR DESCRIPTION
**Pull request by danking**
This is developer only and helpful primarily in dev deploy.
I think its safe as long as we trust hail devs not to
accidentally hit this endpoint.

### CODE REVIEW

**Comment by danking** [comment]
bump

File: ci/ci/ci.py [hunk 1]
```diff
@@ -276,6 +276,20 @@
...
+async def post_update(request, userdata):
+    del userdata
+    log.info(f'developer triggered update')
+
+    async def update_all():
+        for wb in watched_branches:
+            await wb.update(request.app)
+
+    await asyncio.shield(update_all())
+    return web.json_response({})
```
Comment (cseed) [inline ci/ci/ci.py:290]: I prefer `web.Response()` if we're not returning any data.

File: ci/ci/ci.py [hunk 2]
```diff
@@ -276,6 +276,20 @@
     return web.json_response(wb_configs)
 
 
+@routes.post('/api/v1alpha/update')
+@rest_authenticated_developers_only
+async def post_update(request, userdata):
+    del userdata
```
Comment (cseed) [inline ci/ci/ci.py:282]: I prefer the pylint directive because it doesn't have a runtime cost.

File: ci/ci/ci.py [hunk 3]
```diff
@@ -276,6 +276,20 @@
...
+    log.info(f'developer triggered update')
+
+    async def update_all():
+        for wb in watched_branches:
+            await wb.update(request.app)
+
+    await asyncio.shield(update_all())
```
Comment (cseed) [inline ci/ci/ci.py:289]: Do you want this to be synchronous?  `ensure_future` feels more natural here.

**Pull request merged**